### PR TITLE
steam/fhsenv.nix: Add libindicator-gtk2 and libdbusmenu-gtk2

### LINKED ISSUE #181018: Steam tray icon not interactable
### Describe the bug
Left or right clicking the steam tray icon doesn't do anything.

### Steps To Reproduce
1. Launch steam in terminal
2. Right click and left click steam icon in system tray
3. Nothing happens in tray, error in terminal

```
(steam:1945685): LIBDBUSMENU-GLIB-WARNING **: 15:02:00.203: About to Show called on an item wihtout submenus.  We're ignoring it
```

### Expected behavior
Should show tray menu or open steam

### Additional context
Using KDE, full system flake here https://github.com/LunNova/nixos-configs/tree/305102a3f922edb0ae08ed0f8472ad52c7bb9f66/

Seems to match https://github.com/ValveSoftware/steam-for-linux/issues/4795 so may be fixable by adding   `libindicator` `libdbusmenu` and `libappindicator` to the steam FHS env if any are missing.

### Notify maintainers
@jagajaga @jonringer

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.6, NixOS, 22.11 (Raccoon), 22.11.20220708.b39924f`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.9.1`
 - channels(lun): `""`
 - channels(root): `"nixos, nixpkgs"`
 - nixpkgs: `/etc/nix/path/nixpkgs`
```

### PR DESCRIPTION
**Pull request by LunNova**
Fixes #181018

###### Description of changes
Adds two missing libraries to the steam fhs env

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by LunNova** [comment]
```
$ NIXPKGS_ALLOW_UNFREE=1 nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"
   ... SNIP ...
1 package marked as broken and skipped:
steam-run-native

12 packages built:
bottles kodiPackages.steam-launcher lutris minigalaxy playonlinux protontricks steam steam-run steam-tui steamcmd wpsoffice xivlauncher
```

**Review by lightdiscord — APPROVED** [review]
This change makes the tray icon interactive on my end, thanks!

File: pkgs/games/steam/fhsenv.nix [hunk 1]
```diff
@@ -195,6 +195,8 @@ in buildFHSUserEnv rec {
     SDL2_ttf
     SDL2_mixer
     libappindicator-gtk2
+    libindicator-gtk2
+    libdbusmenu-gtk2
```
Comment (winterqt) [inline pkgs/games/steam/fhsenv.nix:199]: Very minor nit that I'm not sure really matters, but maybe these would be better being swapped (so they're alphabetical relative to the other GTK 2 libraries)?

**Review by winterqt — APPROVED** [review]

File: pkgs/games/steam/fhsenv.nix (hunk 1 above)
Comment (SuperSandro2000) [inline pkgs/games/steam/fhsenv.nix:199]:
```suggestion
    libdbusmenu-gtk2
    libindicator-gtk2
```

Comment (LunNova) [reply pkgs/games/steam/fhsenv.nix]: Thanks for doing that for me @SuperSandro2000 !

**Comment by winterqt** [comment]
I think this can be safely backported.

**Pull request merged**